ocp_resources/image_stream.py

```diff
@@ -47,14 +47,18 @@ def to_dict(self):
             {
                 "spec": {
                     "lookupPolicy": {"local": self.lookup_policy},
-                    "tags": {
-                        "from": {"kind": "DockerImage", "name": self.image_repository},
-                        "name": self.tag,
-                        "referencePolicy": {"type": "Source"},
-                    },
+                    "tags": [
+                        {
+                            "from": {
+                                "kind": "DockerImage",
+                                "name": self.image_repository,
+                            },
+                            "importPolicy": {"scheduled": self.scheduled},
+                            "name": self.tag,
+                            "referencePolicy": {"type": "Source"},
+                        }
+                    ],
```

Comment:
please do 
```
if self.scheduled is not None:
            res["spec"]["tags"]["importPolicy"] = self.scheduled
```

Reply:
I'm afraid we have to set it in default false, I missed ["scheduled"] which will cause schedule not working well:
KeyError: 'importPolicy'

Reply:
Must we have `scheduled` under tags always?

Reply:
scheduled is not a must value if we don't need it, but when we set it to false, it is same as without it like:
`      importPolicy: {} `

Reply:
Fixed. Please review again.

Reply:
```
Also tags may have multiple tags but we limit the user to set only one?
```
?

Reply:
Updated the code, please check again @rnetser @myakove